src/tyr/actor.cc

```diff
@@ -313,7 +313,7 @@ actor_t::status(const std::string& request_str, const std::function<void()>* int
   pimpl->set_interrupts(interrupt);
   // parse the request
   Api request;
-  ParseApi(request_str, Options::centroid, request);
+  ParseApi(request_str, Options::status, request);
```

Comment:
I guess we could skip `ParseApi` here and set the `action` directly, but also doesn't hurt too much..

Reply:
We need it for the verbose flag now

Reply:
true..